cityguide/source/app/src/main/java/org/kaaproject/kaa/demo/cityguide/fragment/PlacesFragment.java

```diff
@@ -119,7 +119,7 @@ public void onEvent(Events.ConfigurationUpdated configurationUpdated) {
 
     @Override
     public String getTitle() {
-        return mAreaName + "_" + mCityName;
```

Comment:
@maxml I deleted only "mAreaName", so screen will have a title with a name of current city, for example "Amsterdam", which looks way more beautiful than "Netherlands_Amsterdam"